kuksa-client/setup.cfg

```diff
@@ -1,8 +1,8 @@
 [metadata]
 name = kuksa_client
-author = Sebastian Schildt, Naresh Nayak, Wenwen Chen
-author_email = sebastian.schildt@de.bosch.com, naresh.nayak@de.bosch.com, wenwen.chen@de.bosch.com
-description = kuksa.val python clients and SDK
+author = Eclipse KUKSA Project
+author_email = kuksa-dev@eclipse.org
+description = KUKSA.val Python Client and SDK
```

Comment:
I have no problem dropping `.val` whenever used. Should I possibly update https://github.com/eclipse/kuksa.val/wiki/KUKSA.val-Naming-Guidelines to reflect that?

Reply:
Yes, if we drop it we should change it there too. https://github.com/eclipse/kuksa.val/wiki/KUKSA.val-Naming-Guidelines but htis link does not work for me :/

Reply:
> Yes, if we drop it we should change it there too. https://github.com/eclipse/kuksa.val/wiki/KUKSA.val-Naming-Guidelines but htis link does not work for me :/

That is because I changed the name to https://github.com/eclipse/kuksa.val/wiki/KUKSA-Naming-Guidelines